node/node.go

```diff
@@ -221,14 +221,20 @@ func (node *Node) Beaconchain() *core.BlockChain {
 	return bc
 }
 
-// Add new transactions to the pending transaction list
+// Add new transactions to the pending transaction list.
 func (node *Node) addPendingTransactions(newTxs types.Transactions) {
 	node.pendingTxMutex.Lock()
 	node.pendingTransactions = append(node.pendingTransactions, newTxs...)
 	node.pendingTxMutex.Unlock()
 	utils.GetLogInstance().Debug("Got more transactions", "num", len(newTxs), "totalPending", len(node.pendingTransactions))
 }
 
+// AddPendingTransaction adds one new transaction to the pending transaction list.
+func (node *Node) AddPendingTransaction(newTx *types.Transaction) {
+	node.addPendingTransactions(types.Transactions{newTx})
+	utils.GetLogInstance().Debug("Got ONE more transaction", "totalPending", len(node.pendingTransactions))
```

Comment:
lower case "one"

Reply:
I'm adding in follow-up to fix comments for Richard code as EK is asking - jenkins and travis has been unstable - i want to get this in